cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/AttachmentManager.java

```diff
@@ -190,21 +187,27 @@ public void addAttachment(SQLDatabase db, PreparedAttachment a, BasicDocumentRev
      * Creates a PreparedAttachment from {@code attachment}, preparing it for insertion into
      * the datastore.
      *
+     *
+     * @param attachmentStreamFactory
```

Comment:
I know the method is not public, but probably good to have a small comment for `attachmentStreamFactory` parameter?